python/tvm/topi/cuda/sort.py

```diff
@@ -103,9 +103,9 @@ def sort_ir(data, values_out, axis, is_ascend, indices_out=None):
     nthread_bx = shape[axis] // max_threads + 1
 
     tx = te.thread_axis("threadIdx.x")
-    bx = te.thread_axis("vthread")
+    bx = te.thread_axis("blockIdx.x")
     ib.scope_attr(tx, "thread_extent", nthread_tx)
-    ib.scope_attr(bx, "virtual_thread", nthread_bx)
+    ib.scope_attr(bx, "thread_extent", nthread_bx)
```

Comment:
Doens't seem to make sort faster or more stable.

Reply:
It makes it support dynamic input shapes

Reply:
I'm working on a fuller rewrite of the sort kernel for stabilty in another branch, I understand why this isn't fast or stable, and I can do better for small inputs, but I need to rework things more to support full sized inputs.